docs-markdown/src/test/suite/controllers/include-controller.test.ts

```diff
@@ -14,14 +16,13 @@ const expect = chai.expect;
 
 const root = workspace.workspaceFolders![0].uri;
 const testFile = "../../../../../src/test/data/repo/articles/includes.md";
+const testFilePath = resolve(__dirname, testFile);
 const qpSelectionItems = [
-    { description: root.fsPath + "\\includes", label: "1.md" },
-    { description: root.fsPath + "\\includes", label: "2.md" },
-    { description: root.fsPath + "\\includes", label: "3.md" },
+    { description: root.fsPath + sep + "includes", label: "1.md" },
```

Comment:
Using `path.sep` makes paths OS agnostic.

Reply:
good to know! I'll have to keep that in mind for future changes.